cli.py

```diff
@@ -32,6 +32,9 @@
               help="Toggle token based user authentication.")
 @click.option("--serverurl", default="http://localhost",
               help="Set server url", type=str)
+
+@click.option("--openapi","-o" ,default="doc.jsonld",type=click.File('r'),help="Location to Open API doc")
+
 @click.argument("serve", required=True)
 def startserver(adduser: Tuple, api: str, auth: bool, dburl: str,
                 hydradoc: str, port: int, serverurl: str, token: bool,
```

Comment:
You haven't passed the openapi parameter here

Reply:
im sorry i dont understand , could you please elaborate ?

Reply:
In the startserver function, all the click parameters need to be passed. Don't you think?

Reply:
yes its weird i added it , but somehow its not here , sorry my bad pushing